yellow-paper/docs/decentralization/actors.md

```diff
@@ -0,0 +1,152 @@
+
+:::info Blobs
+Blobs in Ethereum is a special kind of data that is proven to be available at time of publication and for a short period after. After this period (~18 days), the blob is considered shared and can be pruned from the chain. It is not needed to synchronize a new node.
+
+Blobs will likely be stored by certain "super-archive" nodes, but not by all archive nodes. Meaning that it the set of blob-serving nodes likely will be small. As the blob-hash is part of the block header, it is easy to validate that some chunk of data was indeed the published blob. Relies on an 1/n honesty assumption for access to the blob.
+:::
+
+### Actor Types
+
+1. **Mainstream User**:
+   - Currently don't care about technicalities, just want to use the system.
+   - Will likely not run any type of node, unless it is bundled with their wallet so they don't even know it is there.
+   - Generally don't care about trusting Infura or other third parties blindly.
+2. **Enthusiast**: 
+   - More knowledgeable and interested in technology than the mainstream user.
+   - Willing to invest time and resources but not at a professional level.
+   - Likely to run a light node for the extra security.
+3. **Power Users**: 
+   - Technically proficient and deeply engaged.
+   - Likely to have the resources and motivation to run more demanding nodes, and potentially participate in block production.
+4. **Developers**: 
+   - Highly knowledgeable with technical skills.
+   - Interested in detailed state and historical data for development and testing.
+5. **Idealists**: 
+   - Want maximal autonomy, and are willing to invest resources to achieve it.
+   - Will rely entirely on their own nodes and resources for serving their queries, current or historical by running an Archive Node.
+   - Likely to run nodes that contribute directly to the blockchain's operation as part of block production.
+   - Possibly willing to store all blobs
+6. **Institutions**: 
+   - Have significant resources (and potentially technical) capabilities.
+   - Interested in robust participation, and potentially participate in block production for financial gain.
+
+## Aztec Relevant
+
+In the following section, we are following the assumption that Aztec is running on top of Ethereum and using some DA to publish its data.
+
+Beyond the [state](./../state/index.md) of the chain an Aztec node also maintain a database of encrypted data and their tags - an index of tags. The index is used when a user wish to retrieve notes that belong to them. The index is build by the node as it receive new blocks from the network.
+
+While the full index will allow the node to serve all users, it could be configured to only build the index for a subset of the notes. For example, a personal node could be configured to only build the index for the notes that belong to the owner based on their [tag derivation](./../private-message-delivery/note-discovery.md). This would allow the node to serve the owner, but not other users.
```

Comment:
How would the node serve other users? Wouldn't it need to be able to decrypt notes sent to them?

Reply:
I extended the document a bit here